### PR TITLE
Problem #184: BlendingEditor is not rendered by accessing it trough Launchpad.

### PR DESCRIPTION
**Pull request by kegor**
**Problem #184:**
Test-case which leads to fail is described in Screenshots section below.

**Solution:**
We shouldn't make detach from `Parent` (KeptView) for the `OriginalResponse`. 

**Screenshots:**
_Before:_
launchpad_before

### CODE REVIEW

**Comment by kegor** [comment]
Caught flaky problems with WebStuffUpdatingTests.

_[AddNewArea_LoadSameLayoutWithNewArea](https://github.com/Starcounter/Blending/blob/72e2a06271a4a04dd01e90194e7f1f0c74ffa4ad/test/BlendingProvider.Tests/Test/WebStuffUpdatingTests.cs#L38)_:
image

_[RecreateArea_LoadSameLayoutWithRecreatedArea](https://github.com/Starcounter/Blending/blob/72e2a06271a4a04dd01e90194e7f1f0c74ffa4ad/test/BlendingProvider.Tests/Test/WebStuffUpdatingTests.cs#L113)_:
image

File: test/BlendingProvider_AcceptanceHelperOne/Api/MainHandlers.cs [hunk 1]
```diff
@@ -59,7 +61,10 @@ protected void RegisterPageHandlers()
 
             Handle.GET("/BlendingProvider_AcceptanceHelperOne/Form", () =>
             {
-                return Db.Scope(() => new FormPage());
+                Session.Ensure();
```
Comment (un-tone) [inline test/BlendingProvider_AcceptanceHelperOne/Api/MainHandlers.cs]: What the reason to have this here?

File: test/BlendingProvider_AcceptanceHelperOne/Helpers/DataHelper.cs [hunk 2]
```diff
@@ -790,5 +790,35 @@ public bool DeleteTopbarLayoutForMasterWrapperTests()
...
             return isDeleted;
         }
+
+        public void ChangeFormPageTestString()
+        {
+            var newMessage = "Smith";
+
+            Db.Transact(() =>
+            {
+                var formPageData = Db.SQL<FormPageData>("SELECT fpd FROM FormPageData fpd")
```
Comment (un-tone) [inline test/BlendingProvider_AcceptanceHelperOne/Helpers/DataHelper.cs]: Maybe write it shorter?

```cs
var formPageData = Db.SQL<FormPageData>("SELECT fpd FROM FormPageData fpd").FirstOrDefault()
                   ?? new FormPageData();
formPageData.TestInputString = newMessage;
```

File: test/BlendingProvider_AcceptanceHelperOne/Program.cs [hunk 3]
```diff
@@ -15,6 +22,16 @@ static void Main()
             var mainHandlers = new MainHandlers(dataHelper);
 
             mainHandlers.Register();
+
+            Db.Transact(() =>
+            {
+                var formPageData = Db.SQL<FormPageData>("SELECT fpd FROM FormPageData fpd")
```
Comment (un-tone) [inline test/BlendingProvider_AcceptanceHelperOne/Program.cs]: Could be written shorter too:
```cs
var formPageData = Db.SQL<FormPageData>("SELECT fpd FROM FormPageData fpd").FirstOrDefault()
                   ?? new FormPageData();
```

**Pull request merged**